entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -1,294 +1,131 @@
 
-``` powershell
-Add-Migration InitialCreate -Namespace Your.Namespace
-```
+### Install the tools
 
-***
+First, you'll have to install the [EF Core command-line tools](xref:core/miscellaneous/cli/index):
 
-## Update the database
+* We generally recommend using the [.NET Core CLI tools](xref:core/miscellaneous/cli/dotnet), which work on all platforms.
+* If you're more comfortable working inside Visual Studio or have experience with EF6 migrations, you can also use the [Package Manager Console tools](xref:core/miscellaneous/cli/powershell).
 
-Next, apply the migration to the database to create the schema.
+### Create your first migration
 
-### [.NET Core CLI](#tab/dotnet-core-cli)
+You're now ready to add your first migration! Instruct EF Core to create a migration named **InitialCreate**:
 
-```dotnetcli
-dotnet ef database update
-```
-
-### [Visual Studio](#tab/vs)
-
-``` powershell
-Update-Database
-```
-
-***
-
-## Customize migration code
-
-After making changes to your EF Core model, the database schema might be out of sync. To bring it up to date, add another migration. The migration name can be used like a commit message in a version control system. For example, you might choose a name like *AddProductReviews* if the change is a new entity class for reviews.
-
-### [.NET Core CLI](#tab/dotnet-core-cli)
+#### [.NET Core CLI](#tab/dotnet-core-cli)
 
 ```dotnetcli
-dotnet ef migrations add AddProductReviews
+dotnet ef migrations add InitialCreate
 ```
 
-### [Visual Studio](#tab/vs)
+#### [Visual Studio](#tab/vs)
 
 ``` powershell
-Add-Migration AddProductReviews
+Add-Migration InitialCreate
 ```
 
 ***
 
-Once the migration is scaffolded (code generated for it), review the code for accuracy and add, remove or modify any operations required to apply it correctly.
-
-For example, a migration might contain the following operations:
-
-``` csharp
-migrationBuilder.DropColumn(
-    name: "FirstName",
-    table: "Customer");
+EF Core will create a directory called **Migrations** in your project, and generate some files. It's a good idea to inspect what exactly EF Core generated - and possibly amend it - but we'll skip over that for now.
 
-migrationBuilder.DropColumn(
-    name: "LastName",
-    table: "Customer");
+### Create your database and schema
 
-migrationBuilder.AddColumn<string>(
-    name: "Name",
-    table: "Customer",
-    nullable: true);
-```
-
-While these operations make the database schema compatible, they don't preserve the existing customer names. To make it better, rewrite it as follows.
-
-``` csharp
-migrationBuilder.AddColumn<string>(
-    name: "Name",
-    table: "Customer",
-    nullable: true);
+At this point you can have EF create your database and create your schema from the migration. This is as simple as doing the following:
```

Comment:
We've been asked to avoid classifying steps/code as simple, easy, trivial, etc. in our docs. Computer science is hard and making beginners feel stupid isn't the best way to drive adoption. 🙂

Reply:
Will keep it in mind... Although it really does seem very simple :rofl: